velox/expression/SimpleFunctionAdapter.h

```diff
@@ -223,6 +224,14 @@ class SimpleFunctionAdapter : public VectorFunction {
           "Variadic args can only be used as the last argument to a function.");
       auto oneReader = VectorReader<arg_at<POSITION>>(packed, POSITION);
 
+      if constexpr (FUNC::udf_has_callNullFree) {
+        // If is_default_contains_nulls_behavior we return null if the inputs
```

Comment:
is this comment needed here to line 291

Reply:
I simplified this logic at one point and forgot to move the comment.